pyproject.toml

```diff
@@ -48,6 +48,7 @@ dependencies = [
     "django-cors-headers",
     "django-db-mutex",
     "django-debug-toolbar",
+    "django-import-export",
```

Comment:
Ah, and one thing I forgot: You have to execute the update dependency script to pin the version.

Reply:
You did not actually run the script, did you? Just appending the pinned version manually does not work, because the sub-dependencies are missing. `django-import-export` depends on `tablib` and `diff-match-patch` which are not yet installed. And those two might have even more sub-sub-dependencies, so it's always a bad idea to try to generate the pinned versions manually. Just execute the script.

Reply:
We did, and we saw that it introduced changes to about every second line, and thought that raising the versions of all dependencies to their newest available version wasn't really the atomic change we intended to make. We forgot sub-dependencies were a thing… :facepalm:

Reply:
I agree that it's not 100% optimal, but at the moment I don't see a way around it :see_no_evil: 
Writing a bash script which only pins the versions of the newly added dependency is possible but non-trivial I guess. At some point in the future I might have a look at this :grin:

Reply:
I ran it again and pushed the pyproject.toml file as it was updated